src/main/java/use_case/CreateComment/CreateCommentInteractor.java

```diff
@@ -0,0 +1,61 @@
+package use_case.CreateComment;
+
+import entity.*;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+/**
+ * The Creation Comment Interactor
+ */
+public class CreateCommentInteractor implements CreateCommentInputBoundary {
+    private final CreateCommentDataAccessInterface commentDataAccessObject;
+    private final CreateCommentOutputBoundary commentPresenter;
+    private final CommentFactory commentFactory;
+
+    public CreateCommentInteractor(CreateCommentDataAccessInterface commentDataAccessObject,
+                                   CreateCommentOutputBoundary commentPresenter,
+                                   CommentFactory commentFactory) {
+        this.commentDataAccessObject = commentDataAccessObject;
+        this.commentPresenter = commentPresenter;
+        this.commentFactory = commentFactory;
+    }
+
+    @Override
+    public void execute(CreateCommentInputData createCommentInputData) {
+        CommentContent content = new CommentContent(
```

Comment:
If you look in the createComment method in CommentFactory, a CommentContent object is actually created, so we don't need it here.